### DISCUSSION TITLE
NTP Update

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#1988] [Unsupported Versions] [answered]
After modifying /etc/chrony.conf with my ntp servers and restarting my master and sensor node I am getting a handshake failure with elastic.  What can I do to re-initialize the cluster connection and correct the error from changing time.

**Comment by dougburks** [comment, marked as answer]
Do both your master and sensor have the same exact time?

Are they both set to UTC?

When you say `sensor node`, are you referring to a forward node or a heavy node?

Where exactly are you seeing the handshake failure?

**Reply by gcsrellis** [reply]
Initially they are on hardware time when the iso finishes setup. A master node and forward sensor.  After setup completes I edit the ntp and it changes to what we run. I restart both nodes and after opening kibana I get the failures.  Only after does time change does this happen, so I am assuming the ssh connections and everything else involved error out because time has been changed drastically.  Im testing out setting the ntp and network interface up before running the node setup to see if it fixes it.  I assume it will since no services have been setup at that point.

**Reply by dougburks** [reply]
SSH is only used during initial setup, so that shouldn't be an issue.

If it's just a forward node and not a heavy node, then it shouldn't create any handshake failures in Kibana like that.  What is the full text of the handshake failure?

Have you checked the Kibana and Elasticsearch log files on the manager for additional clues?

**Reply by gcsrellis** [reply]
I've looked in /opt/so/log for errors and i cant seem to find anything. When I open a browser though I get dashboard init failed in grafana, I was getting errors in kibana but they seemed to go away but no data. Maybe because the time has surpassed the original time. No clue. I reverted back to a pre node setup snapshot and configured the ntp server and ip before setup and its working fine this way. Ill just add that step for my future deployments.
